packages/chevrotain/api.d.ts

```diff
@@ -1673,6 +1683,10 @@ export interface IParserConfig {
      * For in-depth docs on [Concrete Syntax Trees](http://sap.github.io/chevrotain/docs/guide/concrete_syntax_tree.html):
      */
     outputCst?: boolean
+    /**
+     * Enable computation of CST nodes location
+     */
+    nodePositionTracking?: "full" | "onlyStart" | "onlyOffset" | "none"
```

Comment:
as mentioned elsewhere there is no sense is collecting "onlyStart" location when dealing with CSTNodes
which are often multi lines.

We should always collect pairs (end+start) of the location properties in CSTNode tracking